channeldb/db.go

```diff
@@ -206,6 +206,9 @@ var (
 type DB struct {
 	kvdb.Backend
 
+	// LinkNodeDB separates all DB operations on LinkNodes.
+	LinkNodeDB
```

Comment:
Did not having this cause any issues in practice, or just a drive-by refactor?

Reply:
It's just a drive by refactor to move the channel state step-by-step out of the `channeldb.DB`